dls-controls/malcolmjs
# Feature/info performance #283

## Description

Refactored info Details/Builder/Element so the info object is only rebuilt in a limited number of cases (i.e. if the selected bloc, attribute or sub-element changes) but individual info elements can be provided a path to their value and connect to this in mapStateToProps, minimising unnecessary updates. A value can still be provided in the info object if it is constant for a given attribute (e.g. path).

## Testing instructions

Add a set up instructions describing how the reviewer should test the code
- Review code
- Check Travis build
- Review changes to test coverage
- npm start
- navigate to localhost:3000/PANDA:INENC1/val/.info
- check CPU usage

## Agile board tracking

connect to #283 
closes #283
